events/views.py

```diff
@@ -2557,7 +2557,7 @@ def ajax_state_collage(request):
         tmp = '<option value = None> --------- </option>'
         if collages:
             for i in collages:
-                tmp +='<option value='+str(i.id)+'>'+i.institution_name+'</option>'
+                tmp +='<option value='+str(i.id)+'>'+i.institution_name+', '+i.academic_code+'</option>'
```

Comment:
Minor comment, this voilates PEP8 style guideline, there should be 1 space around operators.